### PR TITLE
Mobile version works

### PR DESCRIPTION
**Pull request by AlaaShaheen2**
added an section for works 
added a new section for about my self
added a gray background for all of the section 
edit on html and css files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+            </li>
+            <li>
+              <img src="images/SeparatorBottom.png" alt="" />
+            </li>
+            <li>
+              <p class="frame-works">Skills</p>
+              <img src="images/union22.png" alt="" class="pic-frame" />
+            </li>
+            <li>
+              <img src="images/SeparatorBottom.png" alt="" />
```
Comment (Lucas-Masaba) [inline index.html:243]: - Please ensure that everything here looks similar to the Figma template. The Font, the icons, the spacing and everything should look familiar
### Your Implementation
image
### Figma
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @AlaaShaheen2  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+}
+
+.proj-info {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-left: 10px;
+  margin-top: 0;
+}
```
[the remark this answers is not in the document]
Comment (AlaaShaheen2) [reply styles.css:223]: @Lucas-Masaba done

File: index.html [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="prog-lang">
+            <li class="lang-styles">html</li>
+            <li class="lang-styles">css</li>
+            <li class="lang-styles">javascript</li>
+          </ul>
+          <button type="button" class="btn">See Projects</button>
+        </section>
```
[the remark this answers is not in the document]
Comment (AlaaShaheen2) [reply index.html:182]: @Lucas-Masaba done

File: index.html [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+            <li>
+              <p class="frame-works">Skills</p>
+              <img src="images/union22.png" alt="" class="pic-frame" />
+            </li>
+            <li>
+              <img src="images/SeparatorBottom.png" alt="" />
+            </li>
+          </ul>
+        </section>
+      </div>
```
[the remark this answers is not in the document]
Comment (AlaaShaheen2) [reply index.html:247]: @Lucas-Masaba done

**Review by wuletawwonte — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @ehma90  👋🏿👋🏿👋🏿, 

Good job implementing the changes requested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

approve

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**